### PR TITLE
React native stock setup

### PR DESCRIPTION
**Pull request by lordmallam**
Tech switch to React-Native

### CODE REVIEW

File: src/components/config/index.js [hunk 1]
```diff
@@ -132,9 +161,10 @@ return this.uniqueRowContent;
 }
```
Comment (andela-rsadiq) [inline src/components/config/index.js]: Can we have lines 87 - 160 extracted into a helper/service? Looks a lot going on in the component already.

File: src/reducers/auth.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ switch (action.type) {
   case Types.PASSWORD_CHANGED:
     return { ...state, password: action.payload, error: '' };
   case Types.LOGIN_SUCCESS:
-    return { ...state, user: action.payload, isLoding: false };
+    return { ...state, user: action.payload, isLoding: false, error: '' };
```
Comment (andela-rsadiq) [inline src/reducers/auth.js]: slight typo here - `isLoading` not `isLoding`...might cause its own errors

File: src/actions/auth.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+      dispatch({ type: Types.LOGIN_FAILED, payload: 'Could not reach server' });
+    }); 
+  }
+};
+
+const getConfigAndSaveUserLocally = (user, dispatch) => {   
+  const localSysDB = new PouchDB(Config.db.localDB_SystemData);
+  const remoteDB = new PouchDB([Config.db.url, Config.db.configDB].join('/')); 
+  const nUser = _.omit(user, '_rev');
+  const URL = [Config.api.url, 'app_config/getconfig', user.name].join('/');
```
Comment (andela-rsadiq) [inline src/actions/auth.js]: string interpolation here on lines 63 and 65.

File: src/config.js [hunk 4]
```diff
@@ -1,20 +1,22 @@
...
-    resources: {
-        logo: require('../src/images/logo.png'),
-        footerNote: 'STOCK v.2.0.1'
-    }
+  api: {
+    url: 'https://dev-lomis.ehealth.org.ng/api',
+    domain: 'https://dev-lomis.ehealth.org.ng'
+    // url: 'http://192.168.10.168:9000/api',
+    // domain: 'http://192.168.10.168:9000'
+  },
```
Comment (andela-rsadiq) [inline src/config.js]: remove unused lines of code.

File: src/actions/auth.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  payload: err
+});
+
+const AuthActions = {
+    loginUser,
+    passwordChanged, 
+    emailChanged,
+    loginFailed
+};
+
```
Comment (lordmallam) [inline src/actions/auth.js]: This is implemented as a handle on all exports from this file because i am using index import/export on the actions folder

**Pull request merged**